File: app/src/main/java/com/my/televip/virtuals/Adapters/DrawerLayoutAdapter.java
package com.my.televip.virtuals.Adapters;

import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import java.util.ArrayList;

import de.robv.android.xposed.XposedHelpers;

public class DrawerLayoutAdapter {

    private final Object drawerLayout;

    public DrawerLayoutAdapter(Object obj){
        drawerLayout = obj;
    }

    public ArrayList<?> getItems(){
        return (ArrayList<?>) XposedHelpers.getObjectField(drawerLayout, Obfuscate.getFieldName("DrawerLayoutAdapter", "items"));
    }

}

File: app/src/main/java/com/my/televip/virtuals/Adapters/DrawerLayoutAdapter.kt
package com.my.televip.virtuals.Adapters

import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers
import java.util.ArrayList

class DrawerLayoutAdapter(private val drawerLayout: Any?) {

    val items: ArrayList<*>?
        get() {
            val target = drawerLayout ?: return null
            return XposedHelpers.getObjectField(target, Obfuscate.getFieldName("DrawerLayoutAdapter", "items")) as? ArrayList<*>
        }
}
